docs/modules/ROOT/pages/tls.adoc

```diff
@@ -47,12 +47,28 @@ Below are the configuration options for the TLS.
 |`secretName`
 | Specifies the name of Kubernetes TLS secret.
 
-|`mutualAuthentication`
+|`mutualAuthentication` (deprecated)
```

Comment:
I think we do not need to keep this value anymore.

Reply:
But, it still exists, we should keep it as deprecated for the old users. Otherwise they will think that their yamls needs to change, however it is backward compatible.

Reply:
It will be listed as deprecated at API reference:
https://docs.hazelcast.com/operator/5.14/api-ref#deprecated-3